src/jail_run.c

```diff
@@ -260,7 +260,7 @@ jail_run(
     if (setgroups(creds->ngroups, creds->groups)) {
         g_propagate_error(error, g_error_new(G_UNIX_ERROR, errno,
             "setgroups error: %s", strerror(errno)));
-    } else if (setresgid(creds->rgid, egid, creds->sgid)) {
+    } else if (setresgid(egid, egid, creds->ruid)) {
```

Comment:
`ruid`? Or did you mean `rgid`?

Reply:
With or without this change `sailjail cat /proc/self/status` prints the same group ids, I'm not seeing any difference.

Reply:
yup, gid, not uid. sloppy me ;-)

The difference comes naturally only with privileged apps. Unless I was dreaming / stte -> firejail does not keep egid intact when executing helpers / the final app binary -> using rgid=egid works around that issue. Fixing firejail would be preferable of course, but that would be probably a sizeable task vs. one liner here.

Reply:
OK, fine. How about adding a comment?

Reply:
fixed uid -> gid + added a FIXME comment